### ISSUE TITLE
[BUG] Panoramic images do not appear properly

### ISSUE THREAD

**Issue by finickyFalcon5** [immich-app/immich#806]
When uploading images from my computer, using the web interface, panoramic image thumbnails appear as a grey box with the image filename (generated by Immich). The images do not open when clicked.

![2022-10-10_15-14](https://user-images.githubusercontent.com/115508507/194960591-2f82ca80-45ce-4277-9750-9cef6240b5ae.png)

**Comment by alextran1502** [comment]
Can you include a picture for testing and debugging?

**Comment by finickyFalcon5** [comment]
I just tried to upload a sample image here and got a warning that the file is too big. So I resampled the image to 25% and uploaded it to Immich and it worked. So the issue is not panoramic images but image size.
I uploaded the original image here: [https://filen.io/d/6e5a00d5-8825-4d61-81fc-34b7ccae2b83#!4sdyQgfOIyYkXJRMmRpzjUAW3fKph8oX](url)
It is 15.1 MB.

**Comment by alextran1502** [comment]
Do you have file size limiting on your reverse proxy?

**Comment by finickyFalcon5** [comment]
I don't know. I downloaded the docker-compose and env files and other than setting the backup location, I just ran the docker-compose file as is.

**Comment by raisinbear** [comment]
Hi, maybe let me add a note here, too. I noticed the same but only when uploading a folder containing many images, including several quite large panoric images, all at once. On the server side, I saw that the microservices container would restart every few minutes afterwards. All I could get out of the logs was a segmentation fault reported by the microservices container. After all in all 8 restarts, all thumbnails for other images had appeared, but not for the largest panoramic images.
Deleting them and readding them one by one worked fine, so I'm still trying to figure out how to reproduce this properly.
